### PR TITLE
Enterprise 1.11.6 release notes

### PR DESCRIPTION
**Pull request by gunnaraasen**
Add release notes for the Enterprise 1.11.6.

_Describe your proposed changes here._

- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1/about-the-project/release-notes.md [hunk 1]
```diff
@@ -19,12 +19,42 @@ InfluxDB Enterprise builds are available. For more information, see
...
 {{% /note %}}
 
+## v1.11.6 {date="2024-08-02"}
+
+### Bug Fixes
+
+- Prevent retention service from hanging, which could lead to a failure to
+  remove old shards.
+- Fix shard precreation service to pre-create shards for more than the first
+  databse.
```
Comment (jstirnaman) [inline content/enterprise_influxdb/v1/about-the-project/release-notes.md:29]:
```suggestion
  database.
```

File: content/enterprise_influxdb/v1/about-the-project/release-notes.md [hunk 2]
```diff
@@ -19,12 +19,42 @@ InfluxDB Enterprise builds are available. For more information, see
...
+- Fix issues with remote iterators caused by floating point literals being
+  rounded in a way that causes lose of precision and can drastically change the
```
Comment (jstirnaman) [inline content/enterprise_influxdb/v1/about-the-project/release-notes.md:31]:
```suggestion
  rounded in a way that causes precision loss and can drastically change the
```

**Review by jstirnaman — APPROVED** [review]
Typo fixes, otherwise 👍

**Review by davidby-influx — CHANGES_REQUESTED** [review]
LGTM with Jason's issues addressed.

**Comment by gunnaraasen** [comment]
I fixed the typos and switched the bullets to inline instead of wrap (to match previous release notes).

I've also added one last note about improved TSM error handling.

**Review by davidby-influx — APPROVED** [review]
LGTM

**Pull request merged**